libs/verifiable-credentials/src/test/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelperTest.java

```diff
@@ -143,28 +205,24 @@ void shouldExtractAgeFromCredentialWithInvalidBirthDate() throws ParseException
     }
 
     @Test
-    void shouldChkIfDocUKIssuedForCredential() throws ParseException {
-        assertTrue(
-                VcHelper.checkIfDocUKIssuedForCredential(
-                        SignedJWT.parse(VC_PASSPORT_NON_DCMAW_SUCCESSFUL_WITH_ICAOCODE)));
-    }
-
-    @Test
-    void shouldCheckIfDocUKIssuedForCredentialWithMissingICAOCode() throws ParseException {
-        assertTrue(
+    void shouldCheckIfDocUKIssuedForCredential() throws ParseException {
+        assertEquals(
+                Boolean.TRUE,
```

Comment:
Can this just stay as `assertTrue`?

Same for the changes below

Reply:
This was a suggested change from a static analysis warning

Reply:
...weird